proto/gateway/service.proto

```diff
@@ -66,6 +66,17 @@ message RegisterEventsResponse {
   map<string, Error> errors = 1;
 }
 
+message TrackRequest {
+  string apikey = 1;
+  string userid = 2;
+  string goalid = 3;
+  string tag = 4;
+  int64 timestamp = 5;
+  double value = 6;
+}
```

Comment:
question:
Is it intentional not to write the query parameters in a snake case to match the rest of the document?

Reply:
Yes.
I wanted to change it to camel or snake case, but it is already running in production, so I kept the original format.